Mu2eKinKal/src/LoopHelixFit_module.cc

```diff
@@ -0,0 +1,369 @@
+//
+// KinKal fit module using the LoopHelix parameterset
+//
+// Original author D. Brown (LBNL) 11/18/20
+//
+// framework
+#include "fhiclcpp/types/Atom.h"
+#include "fhiclcpp/types/Sequence.h"
+#include "fhiclcpp/types/Table.h"
+#include "fhiclcpp/types/Tuple.h"
+#include "art/Framework/Core/EDProducer.h"
+#include "art/Framework/Core/ModuleMacros.h"
+#include "art/Framework/Principal/Event.h"
```

Comment:
The #include for Event.h appears twice.  Please remove the first one - then the file will match the one on Dave's beam2 branch.